modules/tasks/tasks/src/main/java/org/motechproject/tasks/service/TaskTriggerHandler.java

```diff
@@ -176,6 +182,7 @@ private boolean isTaskRetryAlreadyScheduled(Map<String, Object> eventParams) {
     }
 
     private void handleTask(Task task, Map<String, Object> parameters) {
+        metricRegistryService.meter(TASKS_TRIGGER_HANDLER_METER);
```

Comment:
Is it some leftover from testing or are we planning to introuce the metrics module here? Is there some requirement for this? The Task module already exposes the task activity tracking service.